charts/argo-events/templates/argo-events-controller/deployment.yaml

```diff
@@ -108,6 +108,10 @@ spec:
       {{- with .Values.controller.extraContainers }}
         {{- toYaml . | nindent 6 }}
       {{- end -}}
+      {{- with .Values.controller.initContainers }}
```

Comment:
📝 
attribute is in values.yaml https://github.com/argoproj/argo-helm/blob/main/charts/argo-events/values.yaml#L307